### ISSUE TITLE
Bonus fix broke fast-forwarded damage rolls

### ISSUE THREAD

**Issue by draconas1** [EndlesNights/dnd4eBeta#218]
Line 379 of dice.js

```
if(data.bonus){ //stopgap fix because bonus damage type is not being recorded properly
		if(parts[parts.length-1] === "@bonus"){
			let index = data.bonus.lastIndexOf('[');
			if(index >=0) {
				parts[parts.length-1] = '(' + data.bonus.slice(0,index) + ')' + data.bonus.slice(index);
			} else {
				parts[parts.length-1] = '(' + data.bonus + ')';
			}
		}
	} else {
		parts.pop();
	}
```

In the case of a fast-forwarded damage roll (by holding down shift key when clicking the button) `parts` does not have an @bonus entry and `data.bonus == 0 ` which evaluates to false.  This drops into the `else` which pops the last entry from the `parts` array,. which is the roll formula.  Net result is we pass an empty roll into damage and Foundry throws an error when trying to evaluate it.  

I'm away this weekend so can't fix it myself sorry :(

**Comment by EndlesNights** [comment]
Nothing that can't be fixed with a Stopgap v.2 case.

**Comment by EndlesNights** [comment]
Fixed with patch 0.2.81.
